src/main/java/com/spotify/folsom/client/binary/NoopRequest.java

```diff
@@ -27,12 +28,21 @@
 public class NoopRequest extends BinaryRequest<Void> {
 
   public NoopRequest(final int opaque) {
-    super("", opaque);
+    super("X", Charsets.US_ASCII, opaque);
```

Comment:
Will add that!
Something like:
// The key has to be valid to pass the Request contract, but it's not actually used later on.